tests/admin_views/tests.py

```diff
@@ -725,6 +725,11 @@ def test_disallowed_to_field(self):
         response = self.client.get(reverse('admin:admin_views_referencedbyinline_changelist'), {TO_FIELD_VAR: 'name'})
         self.assertEqual(response.status_code, 200)
 
+        # #25622 - Specifying a field of model refered by generic relation shouldn't raise an exception.
```

Comment:
"of a model referred to by a generic"
(and wrap at 79)
You say, "shouldn't raise an exception" but on one of the other tests that returns 400 you say "should raise an exception".
Seems like a candidate to backport to 1.8 if you could add release notes.